src/main/resources/config/config.yaml

```diff
@@ -74,8 +74,11 @@ jpa:
 jerseyClientConfiguration:
   disabledSecureConnection: "false"
 
+emailAddressForRotatingApiTokens: ${EMAIL_ADDRESS_FOR_ROTATING_API_TOKENS}
+
 baseUrl: ${BASE_URL}
 publicApiUrl: ${PUBLICAPI_URL}
+publicAuthUrl: ${PUBLICAUTH_URL}/v1/frontend/auth
```

Comment:
Maybe `publicAuthNewTokenUrl` or something? Just because everywhere else `microserviceNameUrl` is a URL without a path.